core/src/main/scala/org/typelevel/paiges/Doc.scala

```diff
@@ -570,8 +570,11 @@ object Doc {
   /**
    * Invariant: generate is referentially transparent
    */
-  private[paiges] case class LazyDoc(private val generate: () => Doc) extends Doc {
+  private[paiges] class LazyDoc(private val generate: () => Doc) extends Doc {
```

Comment:
Why do we need private val now? Can’t we remove that?

Reply:
Removed.